### ISSUE TITLE
Username/password is not being saved

### ISSUE THREAD

**Issue by OddUnicorn** [Eugeny/tabby#7562]
Version: 1.0.184
Platform: Windows x64 10.0.19045
Plugins: none
Frontend: xterm

I have one login via ssh with username/passwort. Every time I have to enter
the credentials, tabby does not store the credentials, neither through 
settings nor while logging in (checkmark at "remember").

**Comment by buzzy** [comment]
YES! Can't believe they haven't fixed this yet! Also seems like they only save ONE password for all hosts! If I add a new host, it still uses the password for other hosts and if I pick "forget" the password is list for ALL hosts.

**Comment by affinage-digital** [comment]
Same behavior. It saves passwords on most SSH connections, but does not save some.
There are assumptions:
- either weak passwords are not saved
- or Host key fingerprint: ecdsa-sha2-nistp256
